Libraries/xcexecution/Sources/NinjaExecutor.cpp

```diff
@@ -114,6 +114,12 @@ NinjaHash(std::string const &input)
     return ss.str();
 }
 
+static std::string
+NinjaHash(std::string const &input)
```

Comment:
I might have been lazy with this,
Will change callers' signatures to solve this.